extensions/amp-story/1.0/amp-story-bling-link.css

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ amp-story-bling-link {
+  position: relative !important;
+  width: 90px !important;
+  height: 90px !important;
+}
+
+.i-amphtml-story-bling-link-button {
+  position: absolute !important;  
+  border-radius: 50% !important;
+  background: rgba(255, 255, 255, 1) !important;
+  margin: 7.5px !important;
+  width: 75px !important;
+  height: 75px !important;  
+}
+
+amp-story-page[active] .i-amphtml-story-bling-link-pulse {
+  position: absolute !important;
+  border-radius: 50% !important;
+  background: rgba(255, 255, 255, 0.4) !important;
+  margin: 7.5px !important;
+  width: 75px !important;
+  height: 75px !important;
+  animation-name: pulse !important;
+  animation-duration: 1.5s !important;
+  animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1) !important;
+  animation-delay: 0 !important;
+  animation-direction: alternate !important;
+  animation-iteration-count: infinite !important;
+  animation-fill-mode: none !important;
+  animation-play-state: running !important;
```

Comment:
Nit: if the last lines are the default values, you should be able to use `animation: pulse 1.5s cubic-bezier(0.4, 0.0, 0.2, 1) !important;` to save a few lines